src/ONE.Models.CSharp/Enterprise/Twin/OrganizationCategoryConstants.cs

```diff
@@ -0,0 +1,56 @@
+﻿using System;
+
+namespace ONE.Models.CSharp.Enterprise.Twin
+{
+    public static class OrganizationCategoryConstants
+    {
+        public const int Id = 1;
+
+        public static class TenantType
```

Comment:
I might suggest this gets moved to a Constants.TwinTypes namespace

Reply:
Moved and renamed.